ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -67,8 +72,31 @@ - (instancetype)initWithFormatProvider:(id<ORKScaleAnswerFormatProvider>)formatP
     if (self) {
         _formatProvider = formatProvider;
         
-        self.leftRangeLabel.text = [formatProvider localizedStringForNumber:[formatProvider minimumNumber]];
-        self.rightRangeLabel.text = [formatProvider localizedStringForNumber:[formatProvider maximumNumber]];
+        UIView *rightRangeView = nil;
+        UIView *leftRangeView = nil;
+        
+        if ([[formatProvider minimumImage] isKindOfClass:[UIImage class]]) {
```

Comment:
Xcode already provides a warning when assigning incorrect types to properties, so throwing an exception will be redundant. I was merely using the `isKindOfClass:` method as a more robust way to check for `nil`. Since that seems confusing, I will just change it to `[formatProvider minimumImage]`.